packages/manager/modules/pci/src/projects/project/edit/edit.routing.js

```diff
@@ -9,6 +11,79 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('pci_projects_project_parameters'),
+
+      cart: /* @ngInject */ (orderCart, hds) => {
+        if (hds.option && !hds.isCertifiedProject && hds.isValidSupportLevel) {
+          return orderCart
+            .createNewCart(
+              Environment.getUser().ovhSubsidiary,
+              hds.option.planCode,
+            )
+            .then((cart) => orderCart.assignCart(cart.cartId).then(() => cart));
```

Comment:
I'm wondering if the customer should create a cart every time they want to edit their project

Reply:
No, the cart is created only if customer is able to subscribe to HDS.
If had those conditions: `hds.option && !hds.isCertifiedProject && hds.isValidSupportLevel`

Reply:
Ok but what if I respect the conditions and I just want to change my project's name  ? I will create a cart for nothing ?

Reply:
No impact Marie, because we manage the HDS as new section.
So it had a separate CTA for validation.

Reply:
True @marie-j, it will create a new cart.
I implement a workflow to reuse a created cart.

Thanks.